packages/babel-cli/src/babel/index.js

```diff
@@ -212,6 +216,10 @@ if (commander.deleteDirOnStart && !commander.outDir) {
   errors.push("--delete-dir-on-start requires --out-dir");
 }
 
+if (commander.copyDotfiles && !commander.copyFiles) {
+  errors.push("--copy-dotfiles requires --copy-files");
```

Comment:
Is this strictly true? I guess technically you could have `.foo.js` files that you still expect to be compiled and thus don't need dotFiles for.

Maybe what we should do is call it `--include-dotfiles` so if there are dotfiles they get included in whatever operation?